doc/source/_templates/layout.html

```diff
@@ -0,0 +1,11 @@
+<!-- https://github.com/tholman/github-corners -->
+<!-- https://stackoverflow.com/a/53329712/10241324 -->
+{% extends '!layout.html' %}
+{% block document %}
+{{super()}}
```

Comment:
Could we put spaces inside the curly braces? This seems to be a common styling preference for templating and helps make the template more readable.

Reply:
@hodgestar For lines 3-5, how do you want the spaces inserted ? I used an automated html formatter and it didn't add any for these lines.